## [Best genome sequencing strategies for annotation of complex immune gene families in wildlife](https://doi.org/10.1093/gigascience/giac100)

### GigaScience; 2022 (11):giac100
Emma Peel, Luke Silver, Parice Brandies, Ying Zhu, Yuanyuan Cheng, Carolyn J Hogg, Katherine Belov

The biodiversity crisis and increasing impact of wildlife disease on animal and human health provides impetus for studying immune genes in wildlife. Despite the recent boom in genomes for wildlife species, immune genes are poorly annotated in nonmodel species owing to their high level of polymorphism and complex genomic organisation. Our research over the past decade and a half on Tasmanian devils and koalas highlights the importance of genomics and accurate immune annotations to investigate disease in wildlife. Given this, we have increasingly been asked the minimum levels of genome quality required to effectively annotate immune genes in order to study immunogenetic diversity. Here we set out to answer this question by manually annotating immune genes in 5 marsupial genomes and 1 monotreme genome to determine the impact of sequencing data type, assembly quality, and automated annotation on accurate immune annotation.
Genome quality is directly linked to our ability to annotate complex immune gene families, with long reads and scaffolding technologies required to reassemble immune gene clusters and elucidate evolution, organisation, and true gene content of the immune repertoire. Draft-quality genomes generated from short reads with HiC or 10Ã— Chromium linked reads were unable to achieve this. Despite mammalian BUSCOv5 scores of up to 94.1% amongst the 6 genomes, automated annotation pipelines incorrectly annotated up to 59% of manually annotated immune genes regardless of assembly quality or method of automated annotation.
Our results demonstrate that long reads and scaffolding technologies, alongside manual annotation, are required to accurately study the immune gene repertoire of wildlife species.

### [Open Dataset - numbat genome and transcriptome](./species/Myrmecobius_fasciatus.md)
### [Open Dataset - woylie genome and transcriptome](./species/Bettongia_penicillata_ogilbyi.md)

Download data through hyperlink above or using the [AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
  
<br>

This study was supported by the generous contribution of the Presbyterian Ladies' College Sydney and NCRIS funded Bioplatforms Australia. The work was supported by the [Australian Fgenesh++ Service](https://www.biocommons.org.au/fgenesh-plus-plus) provided by the Australian BioCommons and the Pawsey Supercomputing Research Centre.
